examples/get_started/03_figure_element.py

```diff
@@ -0,0 +1,83 @@
+"""
+3. Figure elements
+==================
+The figure below shows the naming of figure elements in PyGMT.
+
+- :meth:`pygmt.Figure()`: having a number of plotting methods. Every new
+  figure must start with the creation of a :meth:`pygmt.Figure()` instance
+- ``frame``: setting plot or map boundaries (a combination of the single 
```

Comment:
Just remove the trailing white spaces to make the Code Style checks happy 🙂.
```suggestion
- ``frame``: setting plot or map boundaries (a combination of the single
```